main.go

```diff
@@ -1,17 +1,21 @@
 package main
 
 import (
-	"log"
-
 	"github.com/stackhound/ande-crawl/crawl"
 	"github.com/stackhound/ande-crawl/db"
+	"log"
+	"strconv"
 )
 
 func main() {
-	records := db.GetAvailableNIS()
+	records, err := db.GetAvailableNIS()
+	if err != nil {
+		log.Fatal("Couldn't get the data:", err)
+	}
 
 	for _, nis := range records {
-		consumption, amount, err := crawl.FetchConsumption(nis)
+		t := strconv.FormatInt(nis.NIS, 10)
+		consumption, amount, pendingBills, expirationDate, err := crawl.FetchConsumption(string(t))
```

Comment:
`strconv.FormatInt` already returns a `string`, see [here](https://golang.org/pkg/strconv/#FormatInt).
This line can be simplified to:
```go
consumption, amount, pendingBills, expirationDate, err := crawl.FetchConsumption(t)
```